contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -188,6 +188,11 @@ var BaseModalView = BaseView.extend({
   		this.$(".modal").modal('hide');
   	}
     this.remove();
+  },
+  closed_modal:function(){
```

Comment:
Addresses problem where closing a modal that has been opened from another modal disables scrolling